vcs/git/clone_update.go

```diff
@@ -45,7 +45,10 @@ func init() {
 }
 
 func Clone(url, dir string, opt vcs.CloneOpt) (vcs.Repository, error) {
-	clopt := git2go.CloneOptions{Bare: opt.Bare}
+	clopt := git2go.CloneOptions{
+		FetchOptions: &git2go.FetchOptions{},
```

Comment:
Originally I thought that as well, but line 61:

```Go
	if rc != nil {
		clopt.RemoteCallbacks = *rc // this line
	}
```

is now a field of the embedded *FetchOptions struct, so it can't be.